.github/ISSUE_TEMPLATE/project-triaging.yaml

```
@@ -0,0 +1,90 @@
+name: Project Triaging
+description: A template for information to look for in an NGI project's resources
+title: <PROJECT_NAME>
```

Add prefix, so we immediately can distinguish this issue from the others.

```suggestion
title: PROJECT: <PROJECT_NAME>
```

---

It makes sense to add a prefix for key results and user stories, but I don't feel there is a need to do the same with projects. The project names should be unique enough to distinguish from normal issues and we'll also have labels for them.

Note: That aside, I don't think the prefix should be in all caps.